lib/srv/db/watcher.go

```diff
@@ -171,7 +171,7 @@ func (s *Server) matcher(resource types.ResourceWithLabels) bool {
 	if !ok {
 		return false
 	}
-	if database.IsRDS() || database.IsRedshift() {
+	if database.IsCloudHosted() {
```

Comment:
This probably needs to be rebased to get https://github.com/gravitational/teleport/pull/11868 fix in.